### PR TITLE
Update 'poetry run' command in README

### PR DESCRIPTION
**Pull request by geetikakay**
## Summary by CodeRabbit

- **Documentation**
	- Added a new section to the README for the `class-generator` project, detailing how to execute the script using Poetry, improving user guidance.

### CODE REVIEW

File: class_generator/README.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ if type class-generator > /dev/null; then eval "$(_CLASS_GENERATOR_COMPLETE=zsh_
 
 ###### Call the script
 
+- Calling the script with poetry
+
+```bash
+poetry run class-generator --kind <kind>
```
Comment (myakove) [inline class_generator/README.md:28]: This is not needed, once running poetry install it will install it system-wide and run it inside poetry.

**Review by myakove — CHANGES_REQUESTED** [review]

**Comment by myakove** [comment]
@coderabbitai review

File: class_generator/README.md (hunk 1 above)
Comment (myakove) [reply class_generator/README.md:28]: What is needed is to change the install command, installation should be done with pipx or pip

using pipx
pipx install openshift-python-wrapper

using pip
python 3 -m pip install openshift-python-wrapper

**Review by myakove — CHANGES_REQUESTED** [review]

File: class_generator/README.md (hunk 1 above)
Comment (geetikakay) [reply class_generator/README.md:28]: ack I

**Comment by myakove** [comment]
@geetikakay already fixed as part of the refactoring of the class-generator, please close this.

**Pull request closed without merging**